data/campaigns/World_Conquest/gui/help_dialog.cfg

```diff
@@ -430,7 +429,9 @@
                                                                                     [/column]
                                                                                     [column]
                                                                                         [label]
-                                                                                            label="↔"
+                                                                                            # wmllint: markcheck off
+                                                                                            label= "↔"
```

Comment:
I'm just not sure what the correct answer is for this one. Making it translatable doesn't seem like a terrible idea, but I don't know if it's a good idea.

Reply:
I search this symbol ("↔")in Wesnoth. It is only translated in zh_CN  po file(wesnoth/po/wesnoth-sota/zh_CH.po.) Since it has been translated should we use translation mark here? or should it be markcheck off?

Reply:
> I'm just not sure what the correct answer is for this one. Making it translatable doesn't seem like a terrible idea, but I don't know if it's a good idea.

I search this symbol ("left_right_arrow")in Wesnoth. It is only translated in zh_CN po file(wesnoth/po/wesnoth-sota/zh_CH.po.) Since it has been translated should we use translation mark here? or should it be markcheck off? Or  should I remove it from zh_ch po file.

Reply:
That's an entirely different campaign, so it has nothing to do with this, but… what is it translated _to_? If it's just translated to the same thing, that may be an indication that it doesn't need to be translated (and there should be a translation mark removed in SotA wherever it's used). If it's translated to something different, that _may_ be an indication that it doesn't need to be translated, but I'd probably want to ask the translator about that.

Reply:
It's not translated in the zh_CN.po file, it's just used in the translators' notes that are also stored in that file.

As for whether this one should be translatable, I'd say "definitely not". If it's to be a translatable string, it needs a `# po:` hint to tell translators what it means in English, and in this case that could be "decoration separating two columns of a table, showing a relationship between the two items that's defined by WC's recruitment mechanics".

Reply:
@stevecotton Thank you, then should I keep the markcheck off here?

Reply:
I'd say to keep the markcheck off here.